packages/react/src/components/ListBox/ListBoxSelection.tsx

```diff
@@ -40,14 +80,14 @@ function ListBoxSelection({
       onClearSelection(event);
     }
   };
-  const handleOnKeyDown = (event) => {
+  const handleOnKeyDown = (event: KeyboardEvent<HTMLDivElement>) => {
     event.stopPropagation();
     if (disabled || readOnly) {
       return;
     }
 
     // When a user hits ENTER, we'll clear the selection
-    if (match(event, keys.Enter)) {
+    if (match(event.code, keys.Enter)) {
```

Comment:
is this currently broken? how do we test this? @tay1orjones

Reply:
Looks like MultiSelect uses <ListBox.Selection/> component.

Reply:
tested and works both ways, so this should be good 👍